v2/session/common-customizations/core/add-ssl-via-nginx.mdx

```diff
@@ -12,36 +12,31 @@ hide_title: true
 This section will guide you through setting up SSL via NGINX so you can query the SuperTokens Core with a secure connection.
 
 :::important
-- This guide assumes you have already installed NGINX on your Ubuntu server.
+- This guide assumes you have already installed NGINX on your server.
 - For the example given below we will be running the SuperToknes core on domain `localhost` and port `3567`
-- The Open API spec for the Core APIs can be found [here](https://app.swaggerhub.com/apis/supertokens/CDI).
 :::
 
 ## 1. Reverse Proxy the SuperTokens Core with NGINX
 
-- The SuperTokens core does not support SSL so we need to use NGINX as a reverse proxy to setup a secure connection.
-
-- We can start by editing the default NGINX site config by running the following command in terminal:
-
-```bash
-sudo nano /etc/nginx/sites-available/default
-```
+The SuperTokens core does not support SSL so we need to use NGINX as a reverse proxy to setup a secure connection.
 
+We can start by opening the default NGINX site config file found at `/etc/nginx/sites-available/default` in a code editor.
```

Comment:
Perhaps we can link to some site that has default location for the file for different OS.. cause this will be diff for diff OSs.